controllers/storage/provision.go

```diff
@@ -28,9 +25,50 @@ const (
 	CredsAccesskey              = "bWluaW8="
 	CredsSecretkey              = "bWluaW8xMjM="
 	DefaultZone                 = "zone-harbor"
-	DefaultMinIOService         = "minio-service"
+	DefaultMinIO                = "minio"
+	DefaultRegion               = "us-east-1"
+	DefaultBucket               = "harbor"
 )
 
+func (m *MinIOReconciler) ProvisionInClusterSecretAsOss(minioInstamnce *minio.MinIOInstance) (*lcm.CRStatus, error) {
+	inClusterSecret := m.generateInClusterSecret(minioInstamnce)
+	err := m.KubeClient.Create(inClusterSecret)
+
+	p := &lcm.Property{
+		Name:  OssSecret,
+		Value: inClusterSecret.Name,
+	}
+	properties := &lcm.Properties{p}
+	return minioReadyStatus(properties), err
+}
+
+func (m *MinIOReconciler) generateInClusterSecret(minioInstamnce *minio.MinIOInstance) *corev1.Secret {
```

Comment:
This is a common method want to create secret？

Reply:
no, just for in-cluster secret